pkg/cli/debug.go

```diff
@@ -449,6 +449,7 @@ func runDebugRaftLog(cmd *cobra.Command, args []string) error {
 
 	start := engine.MakeMVCCMetadataKey(keys.RaftLogPrefix(rangeID))
 	end := engine.MakeMVCCMetadataKey(keys.RaftLogPrefix(rangeID).PrefixEnd())
+	fmt.Printf("Printing keys %s -> %s (%#x - %#x)\n", start, end, start, end)
```

Comment:
Doesn't do what you want -- you need to encode this MVCCMetadata key down to `[]byte` and then print it as hex.

Reply:
i.e. you want the opposite of https://github.com/cockroachdb/cockroach/blob/e46eaeeb01f31bfbcbf05b623361ea2e047ddb6c/pkg/storage/engine/enginepb/decode.go#L52

Not sure if we have it on the Go side.

Reply:
nvm, here it is: https://github.com/cockroachdb/cockroach/blob/50bba75d75e189f6d0496d2a940ee09c1cf82504/pkg/storage/engine/batch.go#L305